jupyter_book/book_template/content/features/notebooks.ipynb

```diff
@@ -97,6 +97,47 @@
     "Note that the image above is captured and displayed by Jekyll."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using popouts\n",
+    "\n",
+    "Sometimes you want to quickly point out an extra piece of information, but\n",
+    "you don't want to break your narrative flow. Use a popout for this!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "popout"
+    ]
+   },
+   "source": [
+    "**For example**, here's some popout content! It was created by adding the\n",
+    "`popout` tag to a cell in the notebook. Jupyter Book automatically converts these\n",
+    "cells into helpful side content."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Add a popout cell by adding `popout` to your cell's tags. Here's what the\n",
+    "cell metadata for a popout cell looks like:\n",
```

Comment:
how do we do this? do we have to edit the raw .ipynb? can this be edited from the notebook itself?